src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.WriteRaw.cs

```diff
@@ -536,7 +536,15 @@ public void WriteRawUtf8LengthGreaterThanOrEqualToIntMax(long len)
                 using MemoryStream ms = new();
                 using Utf8JsonWriter writer = new(ms);
                 ReadOnlySequence<byte> readonlySeq = CreateLargeReadOnlySequence(len);
-                Assert.Throws<ArgumentException>(() => writer.WriteRawValue(readonlySeq));
+
+                if (len == int.MaxValue)
```

Comment:
Per feedback above, this should always throw `ArgumentException`

Reply:
See [comment](https://github.com/dotnet/runtime/pull/76444/files/134cbd9b3f366b1fd9d219163372c9c99c7a3f93..a715f30d106f69e6243c46f82a3cacd12568936d#r1056365655).